ios-base/DroidKaigi 2020/Session/Views/SessionViewController.swift

```diff
@@ -66,7 +69,17 @@ final class SessionViewController: UIViewController {
         let dataSource = SessionViewDataSource()
         let filteredSessions = viewModel.sessions.asObservable()
             .map { sessions -> [Session] in
```

Comment:
Please use `[weak self]` to prevent circular reference 🙏 
For example, 
```swift
.map { [weak self] sessions -> [Session] in
    guard let self = self else { return [] }
```
And you don't need to fix following `self` 👍 

This is preference. So you don't have to fix this.